website/src/components/meetups.ts

```diff
@@ -0,0 +1,150 @@
+  {
+    node: {
+      id: "new-york",
+      name: "GraphQL New York",
+      link: "https://guild.host/graphql-new-york/events",
+      city: "New York",
+      latitude: 40.73,
+      longitude: -73.94,
+    },
+  },
+  {
+    node: {
+      id: "bali",
+      name: "GraphQL Bali",
+      link: "https://guild.host/graphql-bali/events",
+      city: "Bali",
+      latitude: -8.41,
+      longitude: 115.19,
+    },
+  },
+  {
+    node: {
+      id: "taipei",
+      name: "GraphQL Taipei",
+      link: "https://guild.host/graphql-taipei/events",
+      city: "Taipei",
+      latitude: 25.11,
+      longitude: 121.6,
+    },
+  },
+  {
+    node: {
+      id: "20446155",
+      name: "GraphQL TLV",
+      latitude: 32.07,
+      longitude: 34.77,
+      city: "Tel Aviv-Yafo",
+      state: "",
+      country: "il",
+      link: "https://www.meetup.com/graphql-tlv",
+    },
+  },
+  {
+    node: {
+      id: "31704787",
+      name: "GraphQL Zurich",
+      latitude: 47.38,
+      longitude: 8.54,
+      city: "Zürich",
+      state: "",
+      country: "ch",
+      link: "https://www.meetup.com/graphql-zurich",
+    },
+  },
+  {
+    node: {
+      id: "29671383",
+      name: "GraphQL Seattle",
+      latitude: 47.61,
+      longitude: -122.33,
+      city: "Seattle",
+      state: "WA",
+      country: "us",
+      link: "https://www.meetup.com/graphql-seattle",
+    },
+  },
+  {
+    node: {
+      id: "31255122",
+      name: "GraphQL SF",
+      latitude: 37.78,
+      longitude: -122.42,
+      city: "San Francisco",
+      state: "CA",
+      country: "us",
+      link: "https://www.meetup.com/sf-graphql",
+    },
+  },
+  {
+    node: {
+      id: "37102548",
+      name: "GraphQL Gathering EU in Berlin",
+      latitude: 52.52,
+      longitude: 13.38,
+      city: "Berlin",
+      state: "",
+      country: "de",
+      link: "https://www.meetup.com/graphql-gathering-eu",
+    },
+  },
+  {
+    node: {
+      id: "20208222",
+      name: "Amsterdam GraphQL Meetup",
+      latitude: 52.37,
+      longitude: 4.89,
+      city: "Amsterdam",
+      state: "",
+      country: "nl",
+      link: "https://www.meetup.com/amsterdam-graphql-meetup",
+    },
+  },
+  {
+    node: {
+      id: "32869172",
+      name: "Copenhagen GraphQL Meetup Group",
+      latitude: 55.68,
+      longitude: 12.57,
+      city: "Copenhagen",
+      state: "",
+      country: "dk",
+      link: "https://www.meetup.com/copenhagen-graphql-meetup-group",
+    },
+  },
+  {
+    node: {
+      id: "27845373",
+      name: "GraphQL Hamburg",
+      latitude: 53.55,
+      longitude: 10,
+      city: "Hamburg",
+      state: "",
+      country: "de",
+      link: "https://www.meetup.com/graphql-hamburg",
+    },
+  },
+  {
+    node: {
+      id: "20451795",
+      name: "ATX GraphQL",
+      latitude: 30.27,
+      longitude: -97.71,
+      city: "Austin",
+      state: "TX",
+      country: "us",
+      link: "https://www.meetup.com/atx-graphql",
+    },
+  },
+]
```

Comment:
```suggestion
export const meetups = [
  {
    node: {
      id: "london",
      name: "London GraphQL",
      link: "https://guild.host/london-graphql/events",
      city: "London",
      latitude: 51.51,
      longitude: -0.12,
    },
  },
  {
    node: {
      id: "new-york",
      name: "GraphQL New York",
      link: "https://guild.host/graphql-new-york/events",
      city: "New York",
      latitude: 40.73,
      longitude: -73.94,
    },
  },
  {
    node: {
      id: "bali",
      name: "GraphQL Bali",
      link: "https://guild.host/graphql-bali/events",
      city: "Bali",
      latitude: -8.41,
      longitude: 115.19,
    },
  },
  {
    node: {
      id: "bangkok",
      name: "GraphQL Bangkok",
      link: "https://www.meetup.com/graphql-bangkok",
      city: "Bangkok",
      latitude: 13.75,
      longitude: 100.50,
    },
  },
  {
    node: {
      id: "bangalore",
      name: "GraphQL Bangalore",
      link: "https://www.meetup.com/graphql-bangalore",
      city: "Bangkok",
      latitude: 12.97,
      longitude: 77.59,
    },
  },
  {
    node: {
      id: "sydney",
      name: "GraphQL Sydney",
      link: "https://graphql.sydney/",
      city: "Sydney",
      latitude: -33.86,
      longitude: 151.20,
    },
  },
  {
    node: {
      id: "taipei",
      name: "GraphQL Taipei",
      link: "https://guild.host/graphql-taipei/events",
      city: "Taipei",
      latitude: 25.11,
      longitude: 121.6,
    },
  },
  {
    node: {
      id: "20446155",
      name: "GraphQL TLV",
      latitude: 32.07,
      longitude: 34.77,
      city: "Tel Aviv-Yafo",
      state: "",
      country: "il",
      link: "https://www.meetup.com/graphql-tlv",
    },
  },
  {
    node: {
      id: "31704787",
      name: "GraphQL Zurich",
      latitude: 47.38,
      longitude: 8.54,
      city: "Zürich",
      state: "",
      country: "ch",
      link: "https://www.meetup.com/graphql-zurich",
    },
  },
  {
    node: {
      id: "29671383",
      name: "GraphQL Seattle",
      latitude: 47.61,
      longitude: -122.33,
      city: "Seattle",
      state: "WA",
      country: "us",
      link: "https://www.meetup.com/graphql-seattle",
    },
  },
  {
    node: {
      id: "31255122",
      name: "GraphQL SF",
      latitude: 37.78,
      longitude: -122.42,
      city: "San Francisco",
      state: "CA",
      country: "us",
      link: "https://www.meetup.com/sf-graphql",
    },
  },
  {
    node: {
      id: "37102548",
      name: "GraphQL Gathering EU in Berlin",
      latitude: 52.52,
      longitude: 13.38,
      city: "Berlin",
      state: "",
      country: "de",
      link: "https://www.meetup.com/graphql-gathering-eu",
    },
  },
  {
    node: {
      id: "20208222",
      name: "Amsterdam GraphQL Meetup",
      latitude: 52.37,
      longitude: 4.89,
      city: "Amsterdam",
      state: "",
      country: "nl",
      link: "https://www.meetup.com/amsterdam-graphql-meetup",
    },
  },
  {
    node: {
      id: "32869172",
      name: "Copenhagen GraphQL Meetup Group",
      latitude: 55.68,
      longitude: 12.57,
      city: "Copenhagen",
      state: "",
      country: "dk",
      link: "https://www.meetup.com/copenhagen-graphql-meetup-group",
    },
  },
  {
    node: {
      id: "27845373",
      name: "GraphQL Hamburg",
      latitude: 53.55,
      longitude: 10,
      city: "Hamburg",
      state: "",
      country: "de",
      link: "https://www.meetup.com/graphql-hamburg",
    },
  },
  {
    node: {
      id: "20451795",
      name: "ATX GraphQL",
      latitude: 30.27,
      longitude: -97.71,
      city: "Austin",
      state: "TX",
      country: "us",
      link: "https://www.meetup.com/atx-graphql",
    },
  },
]
```

Reply:
Added Bangalore, Bangkok and Sydney. 

FYI most of this data already lives here: https://graphql-events.org/api/meetups

Happy to help you use our API and shape the data, else, if you would like this as the source of truth we can donate/forward graphql-events.org to this page.